db/migrate/20200514091507_make_page_layoutpage_null_false.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class MakePageLayoutpageNullFalse < ActiveRecord::Migration[5.2]
+  def change
+    change_column_null :alchemy_pages, :layoutpage, false
```

Comment:
We need a fourth parameter here that is the default for records with `null` values